onyo/lib/git.py

```diff
@@ -354,65 +329,63 @@ def get_hexsha(self,
 
     def get_commit_msg(self,
                        commitish: str | None = None) -> str:
-        r"""Returns the full commit message of a commit-ish.
+        r"""Return the full commit message of a commit-ish.
 
         Parameters
         ----------
         commitish
             Any identifier that refers to a commit (defaults to "HEAD").
-
-        Returns
-        -------
-        str
-            the commit message including the subject line.
         """
-        return self._git(['log', commitish or 'HEAD', '-n1', '--pretty=%B'])
 
-    def check_ignore(self, ignore: Path, paths: list[Path]) -> list[Path]:
-        r"""Get the `paths` that are matched by patterns defined in `ignore`.
+        return self._git(['log', commitish or 'HEAD', '-n1', '--pretty=%B'])
 
-        This is utilizing ``git-check-ignore`` to evaluate `paths` against
-        a file `ignore`, that defines exclude patterns the gitignore-way.
+    def check_ignore(self,
+                     ignore: Path,
+                     paths: list[Path]) -> list[Path]:
+        r"""Get the subset of ``paths`` that are matched by patterns defined in
+        the ``ignore`` file.
 
         Parameters
         ----------
         ignore
-          Path to a file containing exclude patterns to evaluate.
+            Path to a file containing exclude patterns (in the style of ``.gitignore``).
         paths
-          Paths to check against the patterns in `ignore`.
-
-        Returns
-        -------
-        list of Path
-          Paths in `paths` that are excluded by the patterns in `ignore`.
+            Paths to check
```

Comment:
Normally we still have a dot at the end of these lines.

Reply:
Thanks for the review! I have addressed most issues.

I responded to the `Returns` section question. I feel fairly strongly about the path we have chosen (and is the path chosen by other projects).

The periods I just don't care enough about.